src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -183,23 +253,35 @@ export const AppLevelExternalLinks = ({
                     </TippyCustomized>
                 )}
             >
-                <a
-                    key={linkOption.label}
-                    href={getParsedURL(true, linkOption.value.toString(), details)}
-                    target="_blank"
-                    className="external-link-chip flex left bc-n50 h-24 br-4 cn-7 dc__no-decor dc__border"
-                    rel="noreferrer"
-                >
-                    <img
-                        className="icon-dim-16 mr-4"
-                        src={linkOption.icon}
-                        alt={linkOption.label}
-                        onError={onImageLoadError}
-                    />
-                    <span className="dc__ellipsis-right" data-testid="overview_external_link_value">
-                        {linkOption.label}
-                    </span>
-                </a>
+                <div className="dc__grid br-4 dc__border dc__align-items-center external-link-chip">
+                    <div
```

Comment:
Can we move this to button since it has better semantics and won't have to handle key press 
(enter/space) separately, which we have missed by just adding onClick

Reply:
here gap is 4, where as in button it is 8px by default

Reply:
We can atleast use button tag